### ISSUE TITLE
Support for new version of AppInsights JS SDK

### ISSUE THREAD

**Issue by sowsan** [anastasiia-zolochevska/react-appinsights#38]
Thanks for your great work on this component, is there a support coming up with the latest AppInsights JS SDK : https://github.com/Microsoft/ApplicationInsights-JS/blob/master/AISKU/README.md

**Comment by pviotti** [comment]
Yes, there's a beta version of react-appinsights that you can install with `npm install --save react-appinsights@beta`. It's a complete rewrite of react-appinsights in Typescript. The API is slightly changed: you can find the docs [here](https://github.com/Azure/react-appinsights/blob/master/README.md).

Edit: changed link to reflect recent merge of typescript-rewrite branch into master.

**Comment by pviotti** [comment]
This repository has been archived and the corresponding npm package deprecated. The recommended way of using the new version of Application Insights in a React application is now through its React plugin [@microsoft/applicationinsights-react-js](https://www.npmjs.com/package/@microsoft/applicationinsights-react-js), whose code and documentation reside in the [repository of Application Insights SDK for JavaScript](https://github.com/microsoft/ApplicationInsights-JS). The React plugin is based on version 3.x of react-appinsights mentioned above as beta version of react-appinsights.
